src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -54,6 +60,33 @@ NetworkInfos importCase(UUID caseUuid) {
         return new NetworkInfos(networkUuid, network.getId());
     }
 
+    public ExportNetworkInfos exportCase(UUID networkUuid, String format) {
+        if (!Exporters.getFormats().contains(format)) {
+            throw NetworkConversionException.createFormatUnsupported(format);
+        }
+        MemDataSource memDataSource = new MemDataSource();
+        Network network = networkStoreService.getNetwork(networkUuid);
+        Exporters.export(format, network, null, memDataSource);
+        String extension = getFormatExtension(format);
+        byte[] networkData = memDataSource.getData(extension);
+        String networkName = network.getNameOrId() + extension;
+        return new ExportNetworkInfos(networkName, networkData);
+    }
+
+    String getFormatExtension(String format) {
```

Comment:
This is breaking the plugin architecture of exporters as we should be able to add a new exporter supporter by just by adding it as a dependency in pom.xml.

The export/datasource design is working like this
  - you give a base name to the exporter
  - the exporter is responsible to add the correct extension to each exported file
  - one export could generated SEVERAL files with different extensions (see XIIDM export with .xml + .csv for id dict)

There is an ongoing work to refactor datasources with a new design, but in the meantime we have to support this one.

I didn't had a look at client side, but are you supporting multiple files export?

Using MemDataSource I guess that you can get file names using 
```java
dataSource.listNames(".*)
```

Reply:
No I didn't think about multi-files export. I'll correct that.